OpenTrace.xcodeproj/xcshareddata/xcschemes/opentrace-production.xcscheme

```diff
@@ -31,7 +31,7 @@
       </Testables>
    </TestAction>
    <LaunchAction
-      buildConfiguration = "Release"
+      buildConfiguration = "Debug"
```

Comment:
figured we may as well put this in. as it's really annoying to debug without it and annoying to remember to avoid committing it.